test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -728,7 +728,7 @@ test.describe( 'Image', () => {
 	} );
 } );
 
-test.describe( 'Image - interactivity', () => {
+test.describe.skip( 'Image - interactivity', () => {
```

Comment:
I'm going to skip this test for now because we'll have to adjust the tests in the subsequent PR to use the "new" Lightbox UI.

Reply:
Would be it okay to add a comment here for people like me who forget stuff (like looking at Github annotations) 😄 ?

Reply:
Done 👍